frontend/src/i18n/locales/zh-cn.json

```diff
@@ -306,6 +306,8 @@
       "it": "它",
       "resolves-to": "解析为",
       "is": "是",
+      "is-erc20": "一个 ERC20 代币",
+      "is-erc721": "一个 ERC721 代币",
```

Comment:
@garyghayrat Are these translations okay?

Reply:
Yes!

Reply:
One nit is to just add hyphens in both i18n files, e.g. "ERC-20" and "ERC-721"